### PR TITLE
Update test262 status

### PR DESCRIPTION
**Pull request by ptomato**
Since nobody complained about the last time I updated and replaced the link to a tests PR with the test262 feature flag, let's do it now for all the proposals.

(There isn't really a 1:1 correspondence between tests for one proposal and one pull request, so my hope is that listing the feature flag is more helpful than a link to a PR or search results.)

### CODE REVIEW

**Comment by ljharb** [comment]
Would it be possible to, instead of the check emoji, have just the feature name, as a link that goes somewhere useful inside test262?

**Comment by ptomato** [comment]
Do you have a suggestion for what a useful link might be? The best idea I have so far is a link to a code search for "features" plus the feature name, e.g. https://github.com/tc39/test262/search?q=features+symbols-as-weakmap-keys, but that doesn't seem that useful.

**Comment by ljharb** [comment]
That seems useful enough to me - at least it immediately highlights what it is, that it's a "feature", and what might be included.

**Comment by leobalter** [comment]
Small suggestion: I'd leave the "features" keyword out of the search and have only the search for the actual feature tag.

https://github.com/tc39/test262/search?q=symbols-as-weakmap-keys results are slightly more meaningful.

**Comment by ptomato** [comment]
Odd, those two searches show exactly the same set of results for me, just with slightly different highlighting. I hope GitHub is not showing different results based on an account's search history.

I was thinking "features" would be better for feature tags that are named the same as the string that might show up in code (like "features WeakRef" which omits several irrelevant results that "WeakRef" includes.)

**Comment by ljharb** [comment]
It’s something we can iterate on; I’m content with either choice for now, so we could land the larger refactor :-)

**Comment by leobalter** [comment]
I'm fine either way, I'm not asking anything but suggesting as I'd personally prefer focusing on the highlighted content.

For WeakRef, technically I'd want all the test files containing WeakRef. I'd find it very odd finding a file that uses WeakRef but skips the features tag in Test262.

**Comment by ptomato** [comment]
I'll try to update this today.

FWIW, the files in the WeakRef search that I felt were irrelevant are in docs, harness files, and implementation-contributed tests, as well as [one test](https://github.com/tc39/test262/blob/main/test/built-ins/ShadowRealm/prototype/evaluate/globalthis-available-properties.js) where I'm pretty sure it wouldn't be correct to have WeakRef in the features tag. The first two I realized of course we can skip by restricting the search to JS files, which I'll certainly do. I wish GitHub code search would let us search only in certain paths, to exclude the implementation-contributed and harness files.

**Comment by ptomato** [comment]
This is a good enough start, I think. If the code searches turn out to be annoying, there are other code searches like https://sourcegraph.com/search?q=context:global+repo:tc39/test262+file:test/+%5CbWeakRef%5Cb+lang:JavaScript&patternType=regexp&case=yes

File: README.md [hunk 1]
```diff
@@ -19,23 +19,23 @@ Stage 2 indicates that the committee expects these features to be developed and
...
+| [Resizable and growable ArrayBuffers][resizable]                               | Shu-yu Guo                                                              | Shu-yu Guo                                                              | [resizable-arraybuffer][resizable-tests]         | <sub>[May&nbsp;2021][resizable-notes]</sub>                |
+| [ShadowRealm][shadowrealm]                                                     | Caridy Patiño<br />Jean-Francois Paradis                                | Dave Herman<br />Mark Miller<br />Caridy Patiño<br />Leo Balter<br />Rick Waldron<br />Chengzhong Wu | [ShadowRealm][realms-tests] | <sub>[December&nbsp;2021][realms-notes]</sub>      |
```
Comment (ljharb) [inline README.md:30]: it'd be great to make the feature flag link smaller
```suggestion
| [ShadowRealm][shadowrealm]                                                     | Caridy Patiño<br />Jean-Francois Paradis                                | Dave Herman<br />Mark Miller<br />Caridy Patiño<br />Leo Balter<br />Rick Waldron<br />Chengzhong Wu | <sub>[ShadowRealm][realms-tests]</sub> | <sub>[December&nbsp;2021][realms-notes]</sub>      |
```

**Review by ljharb — APPROVED** [review]
This looks great! Just the one comment.

I'll wait a week or two for others to comment, then merge if there's no objections.

File: README.md (hunk 1 above)
Comment (ptomato) [reply README.md]: Done.

**Comment by ptomato** [comment]
There were no objections so far :smile: 

I've rebased and updated this to the latest status.

**Pull request merged**